### PR TITLE
update csv description

### PR DESCRIPTION
**Pull request by zhiweiyin318**
add annotations and description to csv

### CODE REVIEW

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -29,6 +36,33 @@ spec:
       name: clustermanagers.operator.open-cluster-management.io
       version: v1
   displayName: Cluster Manager
+  description: |
+    The Cluster Manager provides the multicluster hub, which can manage Kubernetes-based clusters across data centers, public clouds, and private clouds. This operator supports the installation and upgrade of Cluster Manager.
```
Comment (pmorie) [inline deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: typo: "of the ClusterManager"

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -29,6 +36,33 @@ spec:
...
+
+    ## How to Install
+    Install Cluster Manager operator by following instructions in top right button `Install`. After installing the operator, create an instance of the ClusterManager resource to install the Cluster Mananger.
+
+    The pods of Cluster Manager are created in the `open-cluster-management-hub` namespace:
+
+    ```
+    $ oc get pods -n open-cluster-management-hub
```
Comment (pmorie) [inline deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: We should probably use `kubectl` in these instructions in the context of this repo.

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -29,6 +36,33 @@ spec:
...
+    NAME                                                       READY   STATUS    RESTARTS   AGE
+    cluster-manager-registration-controller-7c55644568-kms5c   1/1     Running   0          7h13m
+    cluster-manager-registration-controller-7c55644568-mth8f   1/1     Running   0          7h13m
+    cluster-manager-registration-controller-7c55644568-wftnj   1/1     Running   0          7h13m
+    cluster-manager-registration-webhook-77888b7dc4-bwccm      1/1     Running   0          7h13m
+    cluster-manager-registration-webhook-77888b7dc4-qk4tj      1/1     Running   0          7h13m
+    cluster-manager-registration-webhook-77888b7dc4-z9vbp      1/1     Running   0          7h13m
+    ```
+
+    There is a ManagedCluster created in the Hub cluster when the managed cluster is registried to the Hub cluster.
```
Comment (pmorie) [inline deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: typo: registered

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml [hunk 4]
```diff
@@ -29,6 +36,33 @@ spec:
...
+
+    ```
+    $ oc get managedclusters.cluster.open-cluster-management.io
+    NAME                   HUB ACCEPTED   MANAGED CLUSTER URLS     JOINED   AVAILABLE   AGE
+    cluster-dev            true                                    True     True        7h20m
+    cluster-test           false          https://127.0.0.1:8443                        7h25m
+    ```
   icon:
   - base64data: ""
```
Comment (pmorie) [inline deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: @mdelder wdyt about using the open-cluster-management org icon here in the future?

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml [hunk 5]
```diff
@@ -232,9 +266,16 @@ spec:
     type: AllNamespaces
   keywords:
   - open-cluster-management
+  - ClusterManager
+  links:
+  - name: Cluster Manager Operator
+    url: https://github.com/open-cluster-management/registration-operator
   maintainers:
-  - {}
+    - {}
```
Comment (pmorie) [inline deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: There are validations that will demand at least one email address here.

File: deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml [hunk 6]
```diff
@@ -37,6 +44,43 @@ spec:
...
+    ## Prerequisites
+    You need a Hub cluster which has installed Cluster Manager.
+
+    ## How to Install
+    * Install Klusterlet operator by following instructions in top right button `Install`.
+    * Optional: Create an agent namespace used to deploy Klusterlet. The default namespace is `open-cluster-management-agent`.
+    * Create a secret `bootstrap-hub-kubeconfig` with the kubeconfig file of hub cluster in the agent namespace.
+
+      ```
+      $ oc create secret generic bootstrap-hub-kubeconfig --from-file=kubeconfig=<kubeconfig file of hub clsuter> -n <agent namespace>
```
Comment (pmorie) [inline deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml]: kubectl here as well

File: deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml [hunk 7]
```diff
@@ -212,8 +256,16 @@ spec:
     type: AllNamespaces
   keywords:
   - open-cluster-management
+  - Klusterlet
+  links:
+  - name: Klusterlet Operator
```
Comment (pmorie) [inline deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml]: I think I would expect to see `Registration Operator` as the title for this link

**Review by pmorie — COMMENTED** [review]
Thanks for doing this, a couple requested changes below.

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (zhiweiyin318) [reply deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: done

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml (hunk 2 above)
Comment (zhiweiyin318) [reply deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: done

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml (hunk 3 above)
Comment (zhiweiyin318) [reply deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: done

File: deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml (hunk 5 above)
Comment (zhiweiyin318) [reply deploy/cluster-manager/olm-catalog/cluster-manager/0.1.0/cluster-manager.v0.1.0.clusterserviceversion.yaml]: done

File: deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml (hunk 6 above)
Comment (zhiweiyin318) [reply deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml]: done

File: deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml (hunk 7 above)
Comment (zhiweiyin318) [reply deploy/klusterlet/olm-catalog/klusterlet/0.1.0/klusterlet.v0.1.0.clusterserviceversion.yaml]: done

**Pull request merged**